tests/Multiline.elm

```diff
@@ -13,17 +13,50 @@ application =
+                            (Application
+                                (Application
+                                    (Application
+                                        (Application
+                                            (Application
+                                                (Application
+                                                    (Application
+                                                        (Application
+                                                            (Application
+                                                                (Application
+                                                                    (var "fn" { line = 1, column = 0 })
+                                                                    (var "a" { line = 1, column = 0 })
+                                                                    { line = 1, column = 0 }
+                                                                )
+                                                                (var "b" { line = 2, column = 0 })
+                                                                { line = 2, column = 0 }
+                                                            )
+                                                            (var "c" { line = 3, column = 0 })
+                                                            { line = 3, column = 0 }
+                                                        )
+                                                        (var "d" { line = 4, column = 0 })
+                                                        { line = 4, column = 0 }
+                                                    )
+                                                    (var "e" { line = 5, column = 0 })
+                                                    { line = 5, column = 0 }
+                                                )
+                                                (var "f" { line = 6, column = 0 })
+                                                { line = 6, column = 0 }
+                                            )
+                                            (var "g" { line = 7, column = 0 })
+                                            { line = 7, column = 0 }
+                                        )
+                                        (var "h" { line = 8, column = 0 })
+                                        { line = 8, column = 0 }
+                                    )
+                                    (var "i" { line = 9, column = 0 })
+                                    { line = 9, column = 0 }
+                                )
+                                (var "j" { line = 10, column = 0 })
+                                { line = 10, column = 0 }
+                            )
+                            (var "k" { line = 12, column = 1 })
+                            { line = 12, column = 0 }
+                        )
```

Comment:
Apart from the bug of line numbers (Every argument's line being 1 too small apart from the last one) that I assume is coming from Combine dependency rather then this codebase, the isApplication` could be slightly modified to produce this output